torch/onnx/symbolic_helper.py

```diff
@@ -939,6 +939,16 @@ def quantize_helper(g, tensor, scale, zero_point):
     output = g.op("QuantizeLinear", tensor, scale, zero_point)
     return g.op("prim::TupleConstruct", output, scale, zero_point)
 
+# NOTE: Original `bias` is float in PyTorch. Quantization is applied in kernel.
+#       To mimic behavior in ONNX, perform the `bias` quantization step here.
+def requantize_bias_helper(g, bias, input_scale, weight_scale):
+    bias_scale = g.op("Mul", weight_scale, input_scale)
+    bias_zero_point = g.op("Constant", value_t=torch.tensor([0], dtype=torch.int))
+    q_bias = g.op("Cast",
```

Comment:
Are we doing Cast rather than Quantize because Quantize doesn't support int32? If so can you please note that in a comment here?

Reply:
yes, added along the comment above.